scripts/COPYRIGHT-HEADER-SHA

```diff
@@ -0,0 +1 @@
+0867164ffc95e54f04670b5169c09574bdbd9bba
```

Comment:
this isn't great. Should we keep the header somewhere accessible to both repos so we can compute it from the real thing instead of having it copied like this?